### ISSUE TITLE
Bower

### ISSUE THREAD

**Issue by mateusdemboski** [connors/photon#37]
Hi! could be added in the project a bower.json to facilitate the maintenance of "front-end" dependencies. =)

tks,

**Comment by developit** [comment]
People still use Bower?

**Comment by a3bhea** [comment]
"People still use Bower?" ... what? what should i use then??? this is a SINGLE css file with a font, please tell me what should i use? should i decompile it to C++ and rewrite it to PHP just to compile it after that to CoffeeScript which will result in Javascript and "reverse engineer" it back to CSS? This is the single most valuable project that should be in bower after bootstrap right now. I guess you like to write your whole "programs" in javascript and like writing thousands of lines of javascript just to mimic <span>Hello world!</span>. This file (yes its a single file for me) should be in Bower and all the CoffeeScript and React mania should end, its degrading humanity. Thumbs up for the project

**Comment by developit** [comment]
I was going to suggest NPM since it's already available there, no need to take offence.

**Comment by schnti** [comment]
+1

**Comment by sbruchmann** [comment]
@developit FYI, you can already install Photon via npm by using its repository url:

```bash
npm install https://github.com/connors/photon
```

**Comment by developit** [comment]
@sbruchmann indeed. It's a pity the `photon` name is already taken in NPM though :(

**Comment by knownasilya** [comment]
What about namespaced? `npm i @connors/photon`

**Comment by developit** [comment]
@knownasilya yeah, that'd work (as long as nobody is trying to install with npm older than 2.x).

**Comment by knownasilya** [comment]
In that case, it might be worth mentioning the version of npm required.

**Comment by knownasilya** [comment]
Or it could be published as `photonkit`.

**Comment by developit** [comment]
^ nice, that would actually address #36, too ;)

**Comment by neodon** [comment]
I accidentally installed the other photon package with npm and was really confused at first.  Figured out how to add this one with the git url though.  Publishing it on npm (and possibly bower), even under a slightly different name, would be cool. :)

**Comment by dbpolito** [comment]
:+1: for npm support... i know it's easy to use github url, but i think being at npm main repo it would also help the project popularity...

**Comment by wong2** [comment]
`photonkit` is a nice name

**Comment by dbpolito** [comment]
indeed, keep the same name across is good, github, npm, site...

**Comment by mateusdemboski** [comment]
+1 about keep the same name across and using `photonkit`, now used only on site, is possible registry the package on bower and npm...

**Comment by frankhale** [comment]
+1 on Bower package

**Comment by pedzed** [comment]
To @sbruchmann and @gligoran: what is your problem? You can provide a `bower.json` along with a published `package.json`. I am not sure why you are actively downvoting people wanting Bower support.

**Comment by sbruchmann** [comment]
@pedzed I wasn’t voting against `bower` but against the _+1-statement_ of @frankhale since GitHub now supports reactions to comments.

**Comment by gligoran** [comment]
@pedzed Wasn't actually downvoting people for wanting bower support, but the manner in which @a3bhea commented. This is supposed to be a place where people work together, so no need to be condescending.

**Comment by pedzed** [comment]
@sbruchmann I agree, people should not spam with +1's. But it's like slapping a child for doing something wrong: the child does not know what it did wrong, you have to tell him. On the other hand, I understand that you don't want to add _another_ comment just to say that.

@gligoran I apologize if that's the case. The reason I mentioned you was because you downvoted two or three other comments talking about Bower.

I'm currently using https://github.com/connors/photon/issues/37#issuecomment-148870022's suggestion, but it would be nice if it was registered as an NPM/Bower package of course (preferably both).
